module_utils/configuration.py

```diff
@@ -215,6 +215,11 @@ def __init__(self, conn, check_mode=False):
         self._check_mode = check_mode
         self._operation_checker = OperationChecker
 
+        system_info = self._fetch_system_info()
```

Comment:
Let's introduce some lazy loading for the system info as it's not needed for all `add`/`delete`/`get` operations and it requires an extra request on each task. E.g., let's create a `get_build_version` method that will be called from `get_objects_by_filter` and fetch system info if needed.

Reply:
make sense. fixed.